### PR TITLE
Add dimension/file specification note for upload/edit poster

### PR DESCRIPTION
**Pull request by cobbie**

### CODE REVIEW

**Comment by cobbie** [comment]
[Note above the Upload Event Poster portion](https://trello.com/c/x08IeaAc/29-note-above-the-upload-event-poster-portion)

File: client/src/components/routes/events/NewEvent.js [hunk 1]
```diff
@@ -191,6 +191,7 @@ class NewEvent extends Component {
           <UploadArea>
             <label>
               <AppTextP>Upload an Image</AppTextP>
+              <AppTextP size="0.05">NOTE: The image must be an 11 x 11 inch .png file.</AppTextP>
```
Comment (daniddelrio) [inline client/src/components/routes/events/NewEvent.js]: My mistake, it should be 11 inches by 17 inches. Please change accordingly

Comment (daniddelrio) [reply client/src/components/routes/events/NewEvent.js]: Also, to make it clearer, it should be "11 inches by 17 inches" instead of just "11 x 11 inch"``

Comment (cobbie) [reply client/src/components/routes/events/NewEvent.js]: Edited!

**Pull request closed without merging**